modules/oadp-ceph-split.adoc

```diff
@@ -0,0 +1,165 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc
+
+:_content-type: PROCEDURE
+[id="oadp-ceph-split_{context}"]
+= Backing up and restoring data using OADP 1.2 Data Mover and split volumes (CephFS and Ceph RBD)
+
+You can use OpenShift API for Data Protection (OADP) 1.2 Data Mover to back up and restore data in an environment that has _split volumes_, that is, an environment that uses both CephFS and CephRBD.
+
+// [IMPORTANT]
+// ====
+// The CephFS shallow copy feature can only be used for Data Mover backup operations.  The shallow copy volume options are not supported for restore.
+// ====
+
+.Prerequisites
+
+* Ensure a stateful application is running in a separate namespace with PVCs provisioned by both CephFS and CephRBD.
+
+* CephFS is the default `StorageClass` and `VolumeSnapshotClass`.
+
+Procedure
+
+. Create a Data Protection Application (DPA) CR similar to one below. Use the Restic `Secret` you created when you prepared your environment for working with OADP 1.2 Data Mover and Ceph as the value for `spec.features.dataMover.credentialName`. If you do not, then the CR will use the default value `dm-credential` for this parameter.
++
+[NOTE]
+====
+`VolumeOptionsForStorageClass` can be defined for multiple `storageClasses`, thus allowing a backup to complete with volumes with different providers.
```

Comment:
`VolumeOptionsForStorageClass` and `storageClass` need nouns after them. For `storageClass`, make the noun plural instead of the name.
```suggestion
`VolumeOptionsForStorageClass` can be defined for multiple `storageClasses`, thus allowing a backup to complete with volumes with different providers.
```